.github/workflows/sel4test-deploy.yml

```diff
@@ -58,11 +58,8 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        march: [armv7a, armv8a, nehalem]
+        march: [armv7a, armv8a, nehalem, rv32imac, rv64imac]
```

Comment:
We don't have any rv32imac hardware to run and build for, so we should probably leave that one out of the matrix to save on GitHub runners a bit. This reminds me that we should mark the `RISCVVIRT32` platform as `no_hw_build` in the ci-actions repo, because it's a simulation platform only (it is being built and tested in the simulation job).

Reply:
`RISCVVIRT32` has `no_hw_build` set. I just removed this for a test in my branch to check RV32 works as expected.

Reply:
fixed